### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kevin-m-kent**
updating video links for cohort 2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,22 +108,24 @@ Chapter 6 - Functions
 Chapter 7 - Environments
 
 - Cohort 1: [Daryn Ramsden](https://twitter.com/thisisdaryn)  ([slides](https://r4dscommunity.shinyapps.io/environments/) | [static slides](https://r4ds.github.io/bookclub-Advanced_R/Presentations/Week07/Cohort1/Chap7Slides.html#1) | [video](https://www.youtube.com/watch?v=mk7iu1-P8ZU))
-- Cohort 2: [Kevin Kent](https://twitter.com/kevin_m_kent) | ([video](https://www.youtube.com/watch?v=syRMRYKN30k))
+- Cohort 2: [Kevin Kent](https://twitter.com/kevin_m_kent) ([slides](https://github.com/r4ds/bookclub-Advanced_R/blob/main/Presentations/Week07/Cohort2/2020_09_17_advancedR_environments.html) | ([video](https://www.youtube.com/watch?v=syRMRYKN30k))
```
Comment (tanho63) [inline README.md:111]:
```suggestion
- Cohort 2: [Kevin Kent](https://twitter.com/kevin_m_kent) ([slides](https://r4ds.github.io/bookclub-Advanced_R/Presentations/Week07/Cohort2/2020_09_17_advancedR_environments.html) | ([video](https://www.youtube.com/watch?v=syRMRYKN30k))
```

Comment (tanho63) [reply README.md:111]: Directly to GitHub page :)

**Review by tanho63 — CHANGES_REQUESTED** [review]

**Comment by kevin-m-kent** [comment]
ah ok, sounds good. Thanks

**Review by kevin-m-kent — COMMENTED** [review]
looks good to me

**Review by tanho63 — APPROVED** [review]

**Pull request merged**